src/VisualStudio/Core/Def/Implementation/CodeCleanup/AbstractCodeCleanupFixerProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.VisualStudio.Language.CodeCleanUp;
+using Microsoft.VisualStudio.Utilities;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeCleanup
+{
+    internal abstract class AbstractCodeCleanUpFixerProvider : ICodeCleanUpFixerProvider
+    {
+        private readonly ImmutableArray<Lazy<AbstractCodeCleanUpFixer, ContentTypeMetadata>> _codeCleanUpFixers;
+
+        protected AbstractCodeCleanUpFixerProvider(
+            IEnumerable<Lazy<AbstractCodeCleanUpFixer, ContentTypeMetadata>> codeCleanUpFixers)
+        {
+            _codeCleanUpFixers = codeCleanUpFixers.ToImmutableArray();
+        }
+
+        public IReadOnlyCollection<ICodeCleanUpFixer> GetFixers()
+            => _codeCleanUpFixers.SelectAsArray(lazyFixer => lazyFixer.Value);
+
+        public IReadOnlyCollection<ICodeCleanUpFixer> GetFixers(IContentType contentType)
+            => _codeCleanUpFixers.WhereAsArray(handler => handler.Metadata.ContentTypes.Any(contentType.IsOfType))
+                                 .SelectAsArray(l => l.Value);
```

Comment:
why is this SelectAsArray instead of ConvertAll like in the original code?

Reply:
If its C# I don't think I made the change.

Reply:
➡️ This was changed in 0541d2571b282144477c1554815dc854b1c0f4e7